### ISSUE TITLE
Error Path

### ISSUE THREAD

**Issue by Achiyun** [ARM-software/vscode-embedded-debug#24]
When I click on 'run' to download, it will prompt that I cannot find the axf file. For MDK6 I am using in wsl, it will search for the. axf file in the directory of 'win', such as' c:/home/chxx/workspace/', instead of searching in wsl

**Comment by pozitp** [comment]
> When I click on 'run' to download, it will prompt that I cannot find the axf file. For MDK6 I am using in wsl, it will search for the. axf file in the directory of 'win', such as' c:/home/chxx/workspace/', instead of searching in wsl

did you fixed it? i have this error too

**Comment by thegecko** [comment]
Hi @Achiyun @pozitp 

This setup isn't an environment we have tested or support, however we are keen to see if we can resolve the issue.

To confirm, you are using WSL on Windows to use a linux VM. You are connecting to this VM using VS Code on your host and flashing from there and expect the extension to pick up files from the linux container and not the windows host.

**Comment by pozitp** [comment]
> Hi @Achiyun @pozitp 
> 
> This setup isn't an environment we have tested or support, however we are keen to see if we can resolve the issue.
> 
> To confirm, you are using WSL on Windows to use a linux VM. You are connecting to this VM using VS Code on your host and flashing from there and expect the extension to pick up files from the linux container and not the windows host.
> 
> 

Yes, I use WSL on Windows to use a Linux VM (WSL2, Ubuntu LTS). I am connecting through VSCode to VM, and flashing from there. I except pick up files from the linux container.

**Comment by Achiyun** [comment]
> > Hi @Achiyun @pozitp
> > This setup isn't an environment we have tested or support, however we are keen to see if we can resolve the issue.
> > To confirm, you are using WSL on Windows to use a linux VM. You are connecting to this VM using VS Code on your host and flashing from there and expect the extension to pick up files from the linux container and not the windows host.
> 
> Yes, I use WSL on Windows to use a Linux VM (WSL2, Ubuntu LTS). I am connecting through VSCode to VM, and flashing from there. I except pick up files from the linux container.

NO

**Comment by thegecko** [comment]
Thanks for your feedback on this issue. We have identified the problem and a fix requires a considerable architecture change. Please bear with us as we look to schedule this.

**Comment by Achiyun** [comment]
> Thanks for your feedback on this issue. We have identified the problem and a fix requires a considerable architecture change. Please bear with us as we look to schedule this.

Thanks

**Comment by thegecko** [comment]
Update: the latest release has been re-worked so that it should be possible to use it in a WSL or SSH-remote environment (since v1.1.0)
